src/utils/IsMobileState.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @copyright Copyright (c) 2019 Nextcloud community
```

Comment:
please use the original creator of the file :)

Reply:
done :-)